src/index.html

```diff
@@ -11,7 +11,216 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+      class="lines"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="form"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="Surename">Surname:
+            <input
+              type="text"
+              id="Surename"
+              name="Surname"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="Name">
+              Name:
```

Comment:
```suggestion
            Name:
```

Reply:
Not resolved. Just remove 2 spaces before.

Reply:
Check all other similar cases.